universal-login-sdk/lib/api/sdk.ts

```diff
@@ -95,6 +100,14 @@ class UniversalLoginSDK {
     this.deploymentObserver = this.deploymentObserver || new DeploymentObserver(this.blockchainService, this.config!.contractWhiteList);
   }
 
+  async getBalanceObserver(ensName: string) {
```

Comment:
`fetchBalanceObserver` (and also `fetchDeploymentObserver`, `fetchDeploymentReadyObserver`)